showcase/app/controllers/page-components/pagination/index.ts

```diff
@@ -84,74 +102,30 @@ export default class PageComponentsPaginationController extends Controller {
   // -----
   @tracked currentPage_demo2 = 1;
   @tracked currentPageSize_demo2 = 5;
-  @tracked currentSortBy_demo2;
-  @tracked currentSortOrder_demo2;
+  @tracked currentSortBy_demo2: string | undefined = undefined;
+  @tracked currentSortOrder_demo2: HdsTableThSortOrder | undefined = undefined;
   // -----
-  @tracked currentCursor_demo3 = btoa(`next__1`);
+  @tracked currentCursor_demo3: string | null = btoa(`next__1`);
   @tracked currentPageSize_demo3 = 5;
   // -----
   @tracked prevCursor_demo4 = null;
   @tracked nextCursor_demo4 = btoa(`next__1`);
   @tracked currentPageSize_demo4 = 5;
 
-  // =============================
-  // "HOW TO USE" SECTION
-  // =============================
+  noop() {}
 
   get demoRouteName() {
-    // eg. 'page-components.pagination';
-    return this.router.currentRouteName;
+    // eg. 'components.pagination';
+    const routeName = this.router.currentRouteName;
+
+    if (!routeName) return '';
+    return routeName;
```

Comment:
[suggestion] would this work?
```suggestion
    return routeName ?? '';
```